src/main/java/com/thoughtworks/forecastAlexaSkillBackend/cart/CartList.java

```diff
@@ -0,0 +1,33 @@
+package com.thoughtworks.forecastAlexaSkillBackend.cart;
+
+import com.thoughtworks.forecastAlexaSkillBackend.model.ShoppingItem;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartList {
```

Comment:
In case we have more than one User, Singleton cart will be an issue right?

Reply:
Yes.. My quick and dirty suggestion is to have a map of <useremail, cartItems>

Reply:
This can be taken up for enhancement after the MVP @bsneha90 @SwathiVarkala